lib/sles4sap_publiccloud.pm

```diff
@@ -201,6 +201,7 @@ sub sles4sap_cleanup {
 
 sub get_hana_topology {
     my ($self) = @_;
+    $self->run_cmd(cmd => 'cs_wait_for_idle -s 5', timeout => 120);
```

Comment:
Nitpick: use --sleep. We are using --sleep in all other instances and it makes it easier to grep for all uses.